MessageBusReader/Program.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace MessageBusReader
+﻿using MessageBusReader.QueueProcessingHandlers;
```

Comment:
Opinions vary on whether usings should be inside or outside of the namespace. However, please don't do both in the same class.